marklogic-data-hub-central/build.gradle

```diff
@@ -85,6 +91,26 @@ dependencies {
 // We need this to access test resources such as "entity-reference-model" project from core data hub project
 sourceSets.test.resources.srcDir "../marklogic-data-hub/src/test/resources"
 
+// jsonSchema2Pojo is used to generate Java classes that are then used in Swagger annotations for documentation purposes
+// See more at https://github.com/joelittlejohn/jsonschema2pojo/tree/master/jsonschema2pojo-gradle-plugin
+jsonSchema2Pojo {
+    source = files("../specs/models")
+    targetDirectory = file("src/main/java")
+    targetPackage = "com.marklogic.hub.central.schemas"
+}
+
+task deleteSchemaPojos(type: Delete) {
+    delete fileTree("src/main/java/com/marklogic/hub/central/schemas") {
+        include "*.java"
+    }
+}
+generateJsonSchema2Pojo.mustRunAfter deleteSchemaPojos
+
+task generateSchemaPojos {
```

Comment:
So, all the classes in com.marklogic.hub.central.schemas were generated by running the jsonSchema2Pojo gradle task ?

Reply:
Correct. Unfortunately, the generateJsonSchema2Pojo library doesn't add comments to the classes indicating that they were auto-generated, like the ones generated by the ML DS plugin. That's why I added a README.md file to that schemas package.